src/pages/home/report/ReportActionCompose.js

```diff
@@ -502,7 +502,7 @@ ReportActionCompose.defaultProps = defaultProps;
 
 export default compose(
     withWindowDimensions,
-    withDrawerState,
+    // withDrawerState,
```

Comment:
Using this HOC here crashes the app with the following exception. Does anyone know why do we need drawer state here?

```
TypeError: (0, _drawer.useIsDrawerOpen) is not a function. (In '(0, _drawer.useIsDrawerOpen)()', '(0, _drawer.useIsDrawerOpen)' is undefined)

This error is located at:
    in withDrawerState(withNavigationFocus(undefined)) (at withWindowDimensions.js:77)
    in withWindowDimensions(withDrawerState(withNavigationFocus(undefined))) (at withWindowDimensions.js:94)
    in ForwardRef (at ReportView.js:37)
    in RCTView (at View.js:34)
    in View (at SwipeableView/index.native.js:31)
    in SwipeableView (at ReportView.js:36)
    in RCTView (at View.js:34)
    in View (at ReportView.js:32)
    in ReportView (at withOnyx.js:154)
    in withOnyx(ReportView) (at withOnyx.js:178)
    in ForwardRef (at ReportScreen.js:97)
    in RCTView (at View.js:34)
    in View (at ScreenWrapper.js:74)
    in ScreenWrapper (at withNavigation.tsx:26)
    in withNavigation(ScreenWrapper) (at ReportScreen.js:89)
    in ReportScreen (at SceneView.tsx:117)
    in StaticContainer
    in StaticContainer (at SceneView.tsx:110)
    in EnsureSingleNavigator (at SceneView.tsx:109)
    in SceneView (at useDescriptors.tsx:198)
    in RCTView (at View.js:34)
    in View (at Screen.tsx:59)
    in RCTView (at View.js:34)
    in View (at Background.tsx:13)
    in Background (at Screen.tsx:54)
    in Screen (at DrawerView.tsx:231)
    in RCTView (at View.js:34)
    in View (at ResourceSavingView.tsx:42)
    in RCTView (at View.js:34)
    in View (at ResourceSavingView.tsx:37)
    in ResourceSavingScene (at ScreenFallback.tsx:44)
    in MaybeScreen (at DrawerView.tsx:225)
    in RCTView (at View.js:34)
    in View (at ScreenFallback.tsx:31)
    in MaybeScreenContainer (at DrawerView.tsx:191)
    in RCTView (at View.js:34)
    in View (at Drawer.tsx:320)
    in RCTView (at View.js:34)
    in View (at createAnimatedComponent.js:444)
    in AnimatedComponent(View) (at createAnimatedComponent.js:455)
    in ForwardRef(AnimatedComponentWrapper) (at Drawer.tsx:319)
    in RCTView (at View.js:34)
    in View (at createAnimatedComponent.js:444)
    in AnimatedComponent(View) (at createAnimatedComponent.js:455)
    in ForwardRef(AnimatedComponentWrapper) (at Drawer.tsx:310)
    in PanGestureHandler (at Drawer.tsx:301)
    in Drawer (at DrawerView.tsx:256)
    in DrawerViewBase (at DrawerView.tsx:300)
    in RCTView (at View.js:34)
    in View (at DrawerView.tsx:299)
    in SafeAreaProviderCompat (at DrawerView.tsx:298)
    in DrawerView (at createDrawerNavigator.tsx:92)
    in Unknown (at createDrawerNavigator.tsx:91)
    in DrawerNavigator (at MainDrawerNavigator.js:57)
    in MainDrawerNavigator (at withOnyx.js:154)
    in withOnyx(MainDrawerNavigator) (at withOnyx.js:178)
    in ForwardRef (at withWindowDimensions.js:77)
    in withWindowDimensions(Component) (at withWindowDimensions.js:94)
    in ForwardRef (at SceneView.tsx:117)
    in StaticContainer
    in StaticContainer (at SceneView.tsx:110)
    in EnsureSingleNavigator (at SceneView.tsx:109)
    in SceneView (at useDescriptors.tsx:198)
    in RCTView (at View.js:34)
    in View (at CardContainer.tsx:271)
    in RCTView (at View.js:34)
    in View (at CardContainer.tsx:270)
    in RCTView (at View.js:34)
    in View (at CardSheet.tsx:33)
    in ForwardRef(CardSheet) (at Card.tsx:561)
    in RCTView (at View.js:34)
    in View (at createAnimatedComponent.js:165)
    in AnimatedComponent (at createAnimatedComponent.js:215)
    in ForwardRef(AnimatedComponentWrapper) (at Card.tsx:540)
    in PanGestureHandler (at GestureHandlerNative.tsx:13)
    in PanGestureHandler (at Card.tsx:534)
    in RCTView (at View.js:34)
    in View (at createAnimatedComponent.js:165)
    in AnimatedComponent (at createAnimatedComponent.js:215)
    in ForwardRef(AnimatedComponentWrapper) (at Card.tsx:530)
    in RCTView (at View.js:34)
    in View (at Card.tsx:524)
    in Card (at CardContainer.tsx:218)
    in CardContainer (at CardStack.tsx:588)
    in RCTView (at View.js:34)
    in View (at createAnimatedComponent.js:165)
    in AnimatedComponent (at createAnimatedComponent.js:215)
    in ForwardRef(AnimatedComponentWrapper) (at src/index.native.js:94)
    in Screen (at Screens.tsx:37)
    in MaybeScreen (at CardStack.tsx:581)
    in RCTView (at View.js:34)
    in View (at src/index.native.js:123)
    in ScreenContainer (at Screens.tsx:20)
    in MaybeScreenContainer (at CardStack.tsx:501)
    in RCTView (at View.js:34)
    in View (at Background.tsx:13)
    in Background (at CardStack.tsx:499)
    in CardStack (at StackView.tsx:438)
    in SafeAreaProviderCompat (at StackView.tsx:433)
    in RCTView (at View.js:34)
    in View (at StackView.tsx:432)
    in StackView (at createCustomModalStackNavigator.js:19)
    in CustomRootStackNavigator (at AuthScreens.js:172)
    in AuthScreens (at withOnyx.js:154)
    in withOnyx(AuthScreens) (at withOnyx.js:178)
    in ForwardRef (at withWindowDimensions.js:77)
    in withWindowDimensions(Component) (at withWindowDimensions.js:94)
    in ForwardRef (at AppNavigator/index.js:16)
    in AppNavigator (at NavigationRoot.js:46)
    in EnsureSingleNavigator (at BaseNavigationContainer.tsx:424)
    in ForwardRef(BaseNavigationContainer) (at NavigationContainer.tsx:94)
    in ThemeProvider (at NavigationContainer.tsx:93)
    in ForwardRef(NavigationContainerInner) (at NavigationRoot.js:37)
    in NavigationRoot (at Expensify.js:147)
    in Expensify (at withOnyx.js:154)
    in withOnyx(Expensify) (at withOnyx.js:178)
    in ForwardRef (at App.js:9)
    in RNCSafeAreaProvider (at SafeAreaContext.tsx:74)
    in SafeAreaProvider (at App.js:7)
    in App (at renderApplication.js:45)
    in RCTView (at View.js:34)
    in View (at AppContainer.js:106)
    in RCTView (at View.js:34)
    in View (at AppContainer.js:132)
    in AppContainer (at renderApplication.js:39)
```